packages/shared/src/hooks/useOnPostClick.ts

```diff
@@ -65,9 +79,32 @@ export default function useOnPostClick({
         if (!post.read) {
           await incrementReadingRank();
         }
+
+        if (eventName === 'go to link' && feedQueryKey) {
```

Comment:
This condition was the easiest to catch all read post actions and to update feed cache to reflect that post has been read to show `FeedbackCard` for experiment.